server/routes/topics.js

```diff
@@ -1,41 +1,32 @@
 const express = require('express');
-const { path } = require('ramda');
 const Sentry = require('@sentry/node');
 const { logger } = require('../utils/logger');
 
-const fixUrls = element => {
-  const { id, description, href, linkText } = element;
-  switch (element.href) {
-    // TODO: Re-enable when Incentives personalization feature released
-    // case '/content/4204':
-    //   return { id, description, href: '/incentives', linkText };
```

Comment:
Can you shed any light on this @davidthomas22?
It does look hack-y.

Reply:
I think this is related to old designs and is no longer is required